src/common/utils/arrays.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, Type
+
+from domain_classes.blueprint_attribute import BlueprintAttribute
+from domain_classes.dimension import Dimension
+
+
+def create_default_array(
+    dimension: Dimension,
+    blueprint_provider: Callable,
+    create_entity_class: Type[
+        Any
+    ],  # TODO use Type[CreateEntity]. can't use it now due to circular import. Needs refactoring.
+    blueprint_attribute: BlueprintAttribute,
+) -> list:
+    dimensions = dimension.dimensions
+    if dimensions == [""]:
+        raise Exception("This attribute is not an array!")
+    if len(dimensions) == 1:
+        if dimensions[0] == "*":
+            return blueprint_attribute.default if blueprint_attribute.default is not None else []
```

Comment:
This functionality is not tested by the unit tests. Also, I wonder, shouldn't we always use the default value no matter what the dimensions are?

Reply:
Added a simple test for this. 

Yes, Ideally we should use the default value for all dimensions. I added a TODO for that a few lines below, but I can create another issue for this. Should probably not fix it here to avoid increasing the PR size.